.github/workflows/build-test.yml

```diff
@@ -32,6 +32,13 @@ jobs:
       - run: yarn --frozen-lockfile
       - run: yarn allow-scripts
       - run: yarn build
+      - name: Require clean working directory
```

Comment:
Should this go last? In case the other commands result in a dirty working tree as well. They shouldn't, but I can see how it might happen with certain test setups for example, e.g. coverage files.

Reply:
Yeah, this is a good idea. Added!